sympy/physics/mechanics/jointsmethod.py

```diff
@@ -0,0 +1,222 @@
+from sympy.physics.mechanics import Body, Lagrangian, KanesMethod, LagrangesMethod
+from sympy.physics.mechanics.method import _Methods
+
+__all__ = ['JointsMethod']
+
+
+class JointsMethod(_Methods):
+    """Method for formulating the equations of motion using a set of interconnected bodies with joints.
+
+    Parameters
+    ==========
+
+    body : Body or ReferenceFrame
+        The inertial(Newtonion) frame.
+    *joints : Joint
+        The joints in the system
```

Comment:
missing optional method kwarg

Reply:
It is not passed in init, it's being shown in `form_eoms()` docstring.